src/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -418,6 +419,7 @@ private function addElasticsearchSection(ArrayNodeDefinition $rootNode): void
                             ->prototype('scalar')->end()
                         ->end()
                         ->arrayNode('mapping')
+                            ->setDeprecated('api-platform/core', '3.1', sprintf('The "%%node%%" option is deprecated. Configure an %s as $persistenceMeans.', ElasticsearchDocument::class))
```

Comment:
Error? It's the `mapping` option here.

Reply:
The mapping is also deprecated as in fact it's the same thing as what's used in the resource class option...